oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -492,7 +484,8 @@ static ServiceAccountCredentials fromPkcs8(
         serviceAccountUser,
         projectId,
         quotaProject,
-        DEFAULT_LIFETIME_IN_SECONDS);
+        DEFAULT_LIFETIME_IN_SECONDS,
+        false);
```

Comment:
This feature is disabled by default because it doesn't work for all client libraries. Gapic clients will set this flag to `true` to enable the feature.